hack/docs/internal/cronjob-tutorial/api-design.go

```diff
@@ -0,0 +1,129 @@
+package cronjob
```

Comment:
@Eileen-Yu could we have here something like:

`hack/docs/internal/cronjob-tutorial/constants` and then a file name for each part like `controller.go`, `controller_test_code`, `main.go` and etc?